dev/langtool/meta/ckeditor.plugin-about/meta.txt

```diff
@@ -2,8 +2,7 @@
 # For licensing, see LICENSE.md or https://ckeditor.com/license
 
 copy = CKEditor copyright note.
-dlgTitle = Label for the About CKEditor dialog window displayed to Internet Explorer users.
 help = Information about where to find CKEditor documentation.
 moreInfo = Caption for licensing information.
-title = Label for the About CKEditor dialog window.
 userGuide = CKEditor User's Guide title.
+dialogTitle = Label for the About CKEditor dialog window displayed for all browsers.
```

Comment:
Can we change it to:

Label for the About CKEditor 4 dialog window ~displayed for all browsers~.

?

Reply:
Could you also use old `dlgTitle` value? So that we reduce changes amount? And let's put it back between copy and help, in alphabetical order.

After this let's rebuild back the languages so lang files also get the order similar to the previous.